astrodendro/__init__.py

```diff
@@ -1,9 +1,9 @@
 # Licensed under an MIT open source license - see LICENSE
 
-from .dendrogram import Dendrogram, periodic_neighbours
-from .structure import Structure
-from .analysis import ppv_catalog, pp_catalog
-from .plot import DendrogramPlotter
-from .viewer import BasicDendrogramViewer
+from astrodendro.dendrogram import Dendrogram, periodic_neighbours  # noqa
```

Comment:
Why are these `from astrodendro.` while in the next module it's `from . import`?  Is this necessary in `__init__.py` files?  (I wasn't aware)

Reply:
I've reverted this change as I didn't mean to commit it - I think we might want to switch to absolute imports but we could do that in a follow-up